lib/Core/Executor.cpp

```diff
@@ -905,14 +905,21 @@ Executor::fork(ExecutionState &current, ref<Expr> condition, bool isInternal) {
       }
     }
 
+    // We don't add a constraint explicitly yet
```

Comment:
Hi @MartinNowack , I'd like to take a closer look at this patch (BFS turned out to be trickier than expected to implement in the presence of KLEE's state reuse optimization!), but can you quickly clarify this comment?  What is `constraintsAdded` keeping track of?  There are no constraints added here.